scripts/jenkins/ardana/ansible/roles/crowbar_update/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+#
+# (c) Copyright 2019 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+---
+
+- name: 'Check if crowbar product line'
+  fail: msg="the role is intended for crowbar only"
```

Comment:
To keep the same task format as other tasts I would remove the quotes from the task name and call the fail module like:
```
fail:
  msg: "the role is intended for crowbar only"
```